### PR TITLE
Pylint updates

### PR DESCRIPTION
**Pull request by VladimirSlavik**
New Pylint chokes on different stuff than the previous one.

The good part is that there are no new real problems, this is just whack-a-mole with false positives.

### CODE REVIEW

File: tests/nosetests/pyanaconda_tests/modules/boss/modules_test.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def start_one_module_test(self, gio):
         ]
 
         task = StartModulesTask(self._message_bus, service_names, addons_enabled=False)
-        (observer, ) = self._check_started_modules(task, service_names)
+        (observer, ) = self._check_started_modules(task, service_names)  # pylint: disable=unbalanced-tuple-unpacking
```
Comment (poncovka) [inline tests/nosetests/pyanaconda_tests/modules/boss/modules_test.py:67]: This is weird. I think it should be `observers = ` based on other usages of the method.

**Review by poncovka — APPROVED** [review]
Otherwise, it looks good to me. Thanks!

**Review by jkonecny12 — APPROVED** [review]
LGTM! Thanks for fixing this!

**Pull request merged**